portal/api-tests/admin-routes.api-test.js

```diff
@@ -26,14 +26,16 @@ const { get, post } = require('./create-api').createApi(app);
 const email = 'mock email';
 const password = 'mock password';
 const _id = '64f736071f0fd6ecf617db8a';
+const token = '6569ca7a6fd828f925e07c6e';
+const userId = '61e567d7db41bd65b00bd47a';
 let sessionCookie;
 
 describe('user routes', () => {
   beforeEach(async () => {
     login.mockImplementation(mockLogin());
     loginWithSignInLink.mockImplementation(loginWithSignInLinkAsRole(ADMIN));
     sessionCookie = await post({ email, password }).to('/login').then(extractSessionCookie);
-    await get('/login/sign-in-link', { t: '123' }, { Cookie: [sessionCookie] });
+    await get('/login/sign-in-link', { t: token, u: userId }, { Cookie: sessionCookie });
```

Comment:
Why is `sessionCookie` no longer in an array?

Reply:
I thought the Cookie header should just be a string (joined by `;` if you have multiple cookies) but from looking online it does seem like everyone's examples use an array. I'll change it back